Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 
 [dependencies]
 libc = "0.2.66"
-amqp_worker = "0.6.0"
+amqp_worker = {version = "0.6.0", path = "../rs_amqp_worker"}
```

Comment:
Hard coded path!

Reply:
waiting the released of the rs_amqp_worker